src/components/planning/ReasonCheckboxes.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react'
+import {FormField} from "../form-components/FormComponents"
+import styled from "styled-components"
+import {isRequired} from "../form-components/validators/isRequired"
+
+const REASONS = [
+  'Bed en breakfast 2019',
+  'Burgwallenproject Oudezijde',
+  'Corpo-rico',
+  'Digital toezicht Safari',
+  'Digital toezicht Zebra',
+  'Haarlemmerbuurt',
+  'Hotline',
+  'Mystery Guest',
+  'Project Andes',
+  'Project Jordaan',
+  'Project Lobith',
+  'Project Sahara',
+  'Safari',
+  'Safari 2015',
+  'Sahara Adams Suites',
+  'Sahara hele woning',
+  'Sahara meer dan 4',
+  'Sahara Recensies',
+  'Sahara veel adv',
+  'Social Media 2019',
+  'Woonschip (woonboot)',
+  'Zebra'
+]
+
+// TODO add custom checkboxes
+const CheckboxRow = styled.div`
+  input {
+    margin: 10px;
+  }
+`
+
+const ReasonCheckboxes: React.FC = () => <>
+  { REASONS.map((reason, i) => (
+    <CheckboxRow key={reason}>
+      <FormField
+        id={`projects-${i}`}
+        name='projects'
+        component='input'
+        type='checkbox'
+        value={reason}
+        validate={isRequired}
+      />
+      <label htmlFor={`projects-${i}`}>
```

Comment:
Would be less duplication when `id` and `htmlFor` is extracted to `const`

Reply:
Using ASC-label removes the need for id and htmlFor:

```
<Label label='foo'>
   <CheckboxField ... />
</Label>